run.py

```diff
@@ -2,10 +2,13 @@
 
 import gigaset
 import settings
-
-
+from waitress import serve
 
 if __name__ == '__main__':
     gigaset.app.debug = settings.DEBUG
-    gigaset.app.run(host='0.0.0.0', port=settings.PORT)
+    
+    
+    serve(gigaset.app, host="0.0.0.0", port=settings.PORT)
```

Comment:
As mentioned in the other commit, this whole block is intended for debugging only. Please see the documentation of `waitress`, `uwsgi`, or other WSGI servers for details. Personally I was using the following uWSGI config for "production":

```
[uwsgi]
uid = 33
gid = 33
plugin = python3
socket = /var/www/uwsgi/gigaset/gigaset.sock
manage-script-name = true
virtualenv = /var/www/uwsgi/gigaset/venv
chdir = /var/www/uwsgi/gigaset/gigaset
mount = /gigaset=gigaset:app
env = GIGASET_SETTINGS=/var/www/uwsgi/gigaset/settings.py
```

Reply:
reverting the change (except the configurable debug and port configuations.